docs/_includes/ds6/combobox.html

```diff
@@ -0,0 +1,166 @@
+<div id="combobox">
+    <h2><span class="secondary-text">@ebay/skin/</span>combobox</h2>
+    <p>The Skin combobox is intended for use as a custom view (or facade) for the native HTML select element. JavaScript will be required to create 2-way binding between the two.</p>
+    <p><strong>Note:</strong> Because the combobox uses an input element, which must grow to properly display it's contents, we have chosen to have the combobox always remain fluid. It is up to the developer to contain the width, if necessary. Extra care will be needed when placing a combobox side-by-side with other controls such as buttons and textboxes.</p>
+
+    <h3>Default combobox</h3>
```

Comment:
Maybe we change this to "Default Readonly Combobox", and the other subheadings too. Editable combobox will be coming in future!

Reply:
Or we just say in the pre-amble that our current combobox only covers readonly case.

Reply:
I can do both.